gamemodes/terrortown/entities/entities/ttt_traitor_button/init.lua

```diff
@@ -0,0 +1,140 @@
+AddCSLuaFile("shared.lua")
+include("shared.lua")
```

Comment:
Is this line necessary?

Reply:
It's an artifact. If removed, wouldn't it cause issues because the shared.lua isn't loaded?

Reply:
isn't shared.lua loaded by client AND server automatically?

Reply:
I tested it and it sadly needs to be loaded manually :/

Reply:
okay, maybe because of the existant init.lua, who knows...